### ISSUE TITLE
Change 'Settings' to 'Settings file'

### ISSUE THREAD

**Issue by mcpiroman** [microsoft/terminal#1668]
Since a lot of people have issues with what does the "Settings" button, maybe it's better to clarify it by temporarily changing it to "Settings file" or "Open settings file".

**Comment by theterminator3000** [comment]
Or even better a settings window which provides the same options as editing the settings file. I disagree with changing the name though. "Settings file" or "Open settings file" is too wordy. Settings on its own keeps it simple

**Comment by mcpiroman** [comment]
@theterminator3000, Yes of course, there will eventually be a settings window, such as users expect. But for a time it is not, it might be better to call the button after what it is actually doing i.e. opening settings file ( what apparently isn't what users expect, see issues like #1580 or #1664).

**Comment by yidhoe** [comment]
Hey guys \ girls,

This all sounds great, however I would like to add a request to Keep the "Settings File" in the long run too, even if you can't edit it from here.

I manage over 1000 machine and it would be very useful to be able to deploy setting to all of these so they terminals are reconfigured as required.

So for this reason long term I would like to request the ability to configure it by file or through the settings window.

Maybe the there could be an Import \ Export settings option too for those who aren't scripting everything but do want to be able to have a standard set of settings. 

Probably already on your list but just in case.

*Keep up the good work, massive improvement on the current MS Terminal Already! :)

Thanks

Stuart.

**Comment by JushBJJ** [comment]
I'm sure you're talking about #1567

**Comment by yidhoe** [comment]
No actuallky.

Personally the location does not fuss me as it would be scripted anyway, I just want to ensure that the settings remain in a fil we can easily edit and copy between machines in the long run.

As nice as a GUI is having both options is always better. :)

**Comment by JushBJJ** [comment]
Then you probably mean #1564

**Comment by miniksa** [comment]
We're not changing the name of the button right now for the interim state. It's an alpha product. 

Our intent is to have a Settings UI and we will certainly make the file still accessible or more accessible than it is now per the linked issues by @JushBJJ.

Resolving.
